website-custom-views/src/content/getting-started.mdx

```diff
@@ -10,13 +10,13 @@ Before you start development, make sure to have the following:
 
 - A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
 - [Node.js](https://nodejs.org/en/) installed (version `18.x`, recommended `>=20`). Either [`yarn`](https://yarnpkg.com/), [`pnpm`](https://pnpm.io/) or [`npm`](https://www.npmjs.com/get-npm) as a package manager.
-- Basic knowledge of [React](https://reactjs.org/) as well as some experience working with web applications.
+- Basic knowledge of [React](https://reactjs.org/) and some experience working with web applications.
 
-# Installing a starter template
+# Install a starter template
```

Comment:
Nit: if you change these things, we should also change them in the Custom Application docs, as they are copy-pasted.

Or leave them and focus only on the Custom View specific content. The general things will likely be revisited anyway when you work on the unified docs.

This might help you to speed up the reviews.

Reply:
Thanks, @emmenko. I realized this after three files, so let's keep these changes in these 3 files. It's very prominent here but others are not so much.